store/queries/follow_tasks.sql

```diff
@@ -0,0 +1,29 @@
+-- name: EnqueueFollowTask :exec
+INSERT INTO follow_tasks (
+    actor_did,
+    next_try_at,
+    created_at,
+    should_unfollow
+) VALUES ($1, $2, $3, $4);
+
+-- name: GetNextFollowTask :one
```

Comment:
I'd be tempted to put this within a transaction and use SELECT FOR UPDATE -- but I think that can wait until concurrent processing is something we need.

Reply:
Yeah sounds good

Reply:
We have a theoretical processing volume per day of 17,280 tasks, so I think we're fine with sequential for a bit.